source/extensions/omni.isaac.lab/test/sensors/check_imu_sensor.py

```diff
@@ -0,0 +1,195 @@
+# Copyright (c) 2022-2024, The Isaac Lab Project Developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""
+Visual test script for the imu sensor from the Orbit framework.
```

Comment:
Is it not possible to have a unit test for IMU? Check scripts aren't covered in our test runner.